raiden/tests/integration/fixtures/raiden_network.py

```diff
@@ -200,15 +201,16 @@ def raiden_network(
     confirmed_block = raiden_apps[0].raiden.confirmation_blocks + 1
     blockchain_services.deploy_service.wait_until_block(target_block_number=confirmed_block)
 
-    parallel_start_apps(raiden_apps)
+    if start_raiden_apps:
+        parallel_start_apps(raiden_apps)
```

Comment:
Humm, so it looks that we don't need the apps to be started to run `create_all_channels_for_network`. Could you please merge this `if start_raiden_apps` with the other one bellow?

Reply:
unfortunately we do:


https://circleci.com/gh/raiden-network/raiden/70268#tests/containers/3